test/models/MongoDateTimeFormatsSpec.scala

```diff
@@ -24,10 +24,10 @@ class MongoDateTimeFormatsSpec extends SpecBase {
 
   "MongoDateTimeFormats" should {
 
-    //TODO - Result mismatch of res & testReads
-    "read" ignore new Setup {
-      val res = MongoDateTimeFormats.localDateTimeRead
-      res mustBe testReads
+    "read" in new Setup {
+      val testData: JsValue = Json.parse(""" {"$date": 1704228313308}  """)
+      val res = MongoDateTimeFormats.localDateTimeRead.reads(testData)
```

Comment:
It should be done by importing the localDateTimeRead and then using Json.fromJson. Also its better to check the returned object along with whether the Reads was successful.

If it takes time then please add a TODO